R/render-cached-plot.R

```diff
@@ -293,6 +294,7 @@ renderCachedPlot <- function(expr,
   cacheKeyExpr,
   sizePolicy = sizeGrowthRatio(width = 400, height = 400, growthRate = 1.2),
   res = 72,
+  alt = NULL,
```

Comment:
The `alt` arg should be added as the last arg in `plotOutput` and `imageOutput`, so that it doesn't break any existing code that uses positional arguments.